NuRadioReco/framework/particle.py

```diff
@@ -1,26 +1,20 @@
 from __future__ import absolute_import, division, print_function
-import NuRadioReco.framework.parameters as parameters
-import NuRadioReco.framework.parameter_serialization
+from NuRadioReco.framework.parameters import particleParameters as paP
```

Comment:
I'm not a huge fan of obscure abbreviations of classes anyway, and paP looks pretty ugly to me... but I'm aware this is not the only place in the code where this occurs, and `particleParameters.vertex_times` is indeed very long. Won't reject the PR over this but I don't like it ;)